ts/client/src/accounts/group.ts

```diff
@@ -8,6 +8,7 @@ import {
   PublicKey,
 } from '@solana/web3.js';
 import BN from 'bn.js';
+import _ from 'lodash';
```

Comment:
we should import only the functions we need here so that we get the benefits of tree shaking and a smaller bundle size

`import { cloneDeep, merge } from 'lodash';`